whois-api/src/test/java/net/ripe/db/whois/api/rdap/WhoisRdapServiceTestIntegration.java

```diff
@@ -55,21 +55,33 @@
 import static org.junit.jupiter.api.Assertions.assertThrows;
 import static org.junit.jupiter.api.Assertions.fail;
 
-@Tag("IntegrationTest")
+@Tag("ElasticSearchTest")
 @DirtiesContext(classMode = DirtiesContext.ClassMode.AFTER_EACH_TEST_METHOD)
```

Comment:
Why tag needs to be changed?

Reply:
No, we use different tests for ES, please check implementations for abstract class : AbstractElasticSearchIntegrationTest

Reply:
As we talk via Mattermost I compared this class with WhoisRdapElasticServiceTestIntegration and removed all test cases that already exist in the other class. So we don't mix Elastic search tests and common Rdap test in same class.